artemis/src/main/java/sample/camel/SampleJmsRoute.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.camel;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SampleJmsRoute extends RouteBuilder {
+
+   
+    @Override
+    public void configure() throws Exception {
+
+        from("file:src/main/data?noop=true")
+            .to("jms:queue:SCIENCEQUEUE");
+
+       
+        from("jms:queue:SCIENCEQUEUE")
```

Comment:
Maybe `SCIENCEQUEUE` should be in `application.properties` instead of being hard coded, don't you agree?

Reply:
done

spring.artemis.embedded.queue=SCIENCEQUEUE

@Override
    public void configure() throws Exception {

        from("file:src/main/data?noop=true")
            .to("jms:queue:{{spring.artemis.embedded.queue}}");

       
        from("jms:queue:{{spring.artemis.embedded.queue}}")
            .log("${body}");
    }